tools/stress-cluster/cluster/README.md

```diff
@@ -78,13 +78,13 @@ Avoid checking in the updated dev values, they are for local use only.
 <tools repo>/eng/common/scripts/stress-testing/deploy-stress-tests.ps1 -Login -Environment dev
 ```
 
-## Test Cluster
+## Playground Cluster
 
-The test cluster is the main ad-hoc cluster made available to SDK developers and partners. Changes to this cluster
+The playground cluster is the main ad-hoc cluster made available to SDK developers and partners. Changes to this cluster
 should be made carefully and announced in advance in order not to disrupt people's work.
 
 ```
-./provision.ps1 -env test
+./provision.ps1 -env pg
```

Comment:
Provision script also needs a change here.

Reply:
What do you mean by provision script change. The provision script takes in $Environment parameter and just use those, I don't think there's any hard coded "test cluster" things.

Reply:
Followup in #3491